crates/meroctl/src/cli/context/alias.rs

```diff
@@ -71,8 +66,13 @@ impl ContextAliasCommand {
                     return Ok(());
                 }
 
-                let lookup_result =
-                    lookup_alias(&connection.api_url, auth_key, alias, None).await?;
+                let lookup_result = lookup_alias(
+                    &connection.api_url,
+                    connection.auth_key.as_ref().unwrap(),
```

Comment:
this is worse that what you had before, it's still expecting the auth key to be present only now it's panicking instead of returning an error

it should be the other way around.. `lookup_alias` should expect an `Option`